lib/galaxy/model/__init__.py

```diff
@@ -2788,7 +2789,9 @@ class Notification(Base, Dictifiable, RepresentById):
     variant = Column(
         String(16), index=True
     )  # Defines the 'importance' of the notification ('info', 'warning', 'urgent', etc.). Used for filtering, highlight rendering, etc
-    content = Column(JSONType)  # Structured content in JSON. Depending on the category of the notification
+    # A bug in early 23.1 led to values being stored as json string, so we use this special type to process the result value twice.
```

Comment:
Notifications will be deleted from the DB after a few weeks. Should we add a comment here that this should be changed in the next version?

Reply:
expiration time is not required, so I don't think we can count on that, and e.g. shared with notifications should probably not expire automatically.

Reply:
Yeah, we cannot rely on the expiration time as Marius said. But this solution is flexible and doesn't require a db migration or other intrusive fix :+1: